python-threatexchange/threatexchange/extensions/vpdq/video_vpdq.py

```diff
@@ -63,7 +63,7 @@ def validate_signal_str(cls, signal_str: str) -> str:
         vpdq_hashes = json_to_vpdq(signal_str)
         last_frame_number = -1
         for hash in vpdq_hashes:
-            if not re.match("^[0-9a-f]{64}$", hash.hex):
+            if not re.match("^[0-9a-f]{64}$", str(hash.hex, "utf-8")):
```

Comment:
It is because the hex_str created by the vpdq package is actually c_string. It needs to be decoded first and then uses re-match. I wasn't sure where I should implement this decode, because all the hash/distancing in vpdq are done in C++ library. But for easier usage, only having c_string as an internal usage within vpdq package would be better. Any suggestions on this?